tests/misc/src/test_exprs.rs

```diff
@@ -0,0 +1,26 @@
+//! feature_core_intrinsics,
```

Comment:
Why do we need `#![feature(core_intrinsics)]` for this?

Reply:
I think it does need `//! feature_label_break_value`, though, as `#![feature(label_break_value)]` was only recently stabilized and we're still stuck on an older `rustc`.

Reply:
We don't--it was a holdover from copy-pasting another test case. Removed.

Reply:
Added `feature_label_break_value` for the `sizeof` test. I'm not sure why that one wasn't failing in CI prior to now.

Reply:
Yeah, that's weird, but I guess if it's fixed now it's all good.